### PR TITLE
Add sequential tests to Makefile for easier use

### PR DESCRIPTION
**Pull request by dylanlott**
This adds another target to our makefile that runs tests 5 times in a row with caching turned off, but failfast turned on. 

Eventually I think we should consider making this the main test command for our Travis tests to help prevent flakiness. 

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -112,6 +112,10 @@ test-all-in-one: ## Test docker images locally
 	&& $(MAKE) satellite-image storagenode-image gateway-image \
 	&& ./scripts/test-aio.sh
 
+.PHONY: test-sequential
+test-sequential: ## Run tests five times with caching turned off  and failfast
+	seq 1 5 | xargs -I{} sh -c 'go test -v ./... -count=1 -race -failfast'
```
Comment (wthorp) [inline Makefile]: why can't you use -count=5 instead?

**Comment by brimstone** [comment]
Why do the tests need to be run multiple times?
And why 5 times and not 2 or 10 or 9001?

**Comment by dylanlott** [comment]
> Why do the tests need to be run multiple times?
> And why 5 times and not 2 or 10 or 9001?

I was going to make it take an argument, so that the number of times could be configured.

As for why we want to run it multiple times - We've talked about making sure our tests are not racy or flakey. This would help with that, but would increase time for tests to run. @jtolds has mentioned in the past that we might want to try this. I was using it a lot for my local tests, so it seemed reasonable to make a target for it. I wouldn't be opposed to making our existing `make test` target take an optional param for a number of times, either. I'm not a make wizard like you are, would love a better way to do that.

File: Makefile (hunk 1 above)
Comment (dylanlott) [reply Makefile]: You are 100% right, I'll update this to take that.

**Comment by egonelbre** [comment]
Note once we have tests running on multiple platforms we will have the "repeated tests" thing effectively for free. But, this means we need actually need to have multiple platforms.

**Comment by egonelbre** [comment]
I think this won't be needed, since we cannot run this currently without bogging down Travis server (for specific tests you usually want to specify the test you want to rerun). Alternatively, when we have Jenkins we get repeated tests by testing on multiple platforms.

**Pull request closed without merging**